src/app/tests/TestReadInteraction.cpp

```diff
@@ -1841,6 +1843,148 @@ void TestReadInteraction::TestSubscribeWildcard(nlTestSuite * apSuite, void * ap
     NL_TEST_ASSERT(apSuite, ctx.GetExchangeManager().GetNumActiveExchanges() == 0);
 }
 
+// Subscribe (wildcard, C3, A1), then setDirty (E2, C3, wildcard), receive one attribute after setDirty
```

Comment:
We should really be adding tests for `AttributePathParams::Intersects` since that drives the crux of a lot of this stuff...

Reply:
Yes. I have written tests for intersects locally. Would push the test in branch this night. Thanks

Reply:
Done, thanks